mobileapp/README.md

```diff
@@ -5,14 +5,13 @@ This is a capacitor project that builds the mobile apps from the web directory
 ## Prerequisites
 
 - Follow this guide: https://capacitorjs.com/docs/getting-started/environment-setup (but skip the Android SDK part)
```

Comment:
Could also be worth mentioning ruby versions and which ruby version manager to use?

Reply:
That makes sense, my setup is like this:

```suggestion
- Setup Ruby environment: Use [`rbenv`](https://github.com/rbenv/rbenv) to manage versions, and make sure you install and set `3.1.2` as the default
- Follow this guide: https://capacitorjs.com/docs/getting-started/environment-setup (but skip the Android SDK part)
```